packages/ios_platform_images/lib/ios_platform_images.dart

```diff
@@ -78,7 +113,14 @@ class IosPlatformImages {
   ///
   /// See [https://developer.apple.com/documentation/uikit/uiimage/1624146-imagenamed?language=objc]
   static FutureMemoryImage load(String name) {
```

Comment:
I'm wondering if you need to export the `FutureMemoryImage` class at all if this is the main way to construct it. Would it make sense to just return an `ImageProvider` here?

Reply:
Done